ui/src/pages/namespace/Explorer/Service/ServiceEditor/Form/index.tsx

```diff
@@ -121,10 +124,14 @@ export const Form: FC<FormProps> = ({ defaultConfig, children }) => {
               </SelectTrigger>
               <SelectContent>
                 <SelectGroup>
-                  <SelectLabel>Size</SelectLabel>
-                  <SelectItem value="small">small</SelectItem>
-                  <SelectItem value="medium">medium</SelectItem>
-                  <SelectItem value="large">large</SelectItem>
+                  <SelectLabel>
+                    {t("pages.explorer.service.editor.form.size.label")}
+                  </SelectLabel>
+                  {sizeOptions.map((value, index) => (
+                    <SelectItem key={index} value={value}>
+                      {value}
+                    </SelectItem>
```

Comment:
Since all options are unique, I think we can simplify this mapping function a bit and don't need to use the index as the key, but the value itself. I would also give the variable a slightly less abstract name

```typescript
{sizeOptions.map((size) => (
   <SelectItem key={size} value={size}>
    {size}
  </SelectItem>
))}
```

Reply:
I agree, that's really nice! 🙂